component/main.jsonnet

```diff
@@ -66,10 +68,23 @@ local readServices = kube.ClusterRole('appcat:services:read') + {
   ],
 };
 
+// adding namespace for syn-appcat
+local ns = kube.Namespace('syn-appcat');
```

Comment:
We should probably need to make it configurable in defaults like any other namespace.

Reply:
Fixed